Dockerfile

```diff
@@ -4,6 +4,7 @@ WORKDIR /app
 RUN adduser -D -h /home/term -s /bin/sh term && \
   echo "term:term" | chpasswd
 EXPOSE 3000
+RUN apk add --update build-base python openssh
 COPY . /app
-RUN apk add --update build-base python openssh && yarn
+RUN yarn
```

Comment:
Id rather this kept as one line

Reply:
The point was to be able to reuse caching when developing and testing using the Docker image.
Before every time I wanted to test, it would re-do all the apk operations before the step just before (`COPY . /app`) changed.

With the new sequence the `apk` part does not have to re-run every time the image with rebuilt with new app changes. This is an improvement on the dev workflow, and I though the addition of one layer was worth the trade-off, but I can remove that change if you still don't agree with it.

Reply:
Better solution move the `apk` to be part of the `RUN` on line 4

Reply:
Sure that would work, let me test and update the patch.